### DISCUSSION TITLE
Change train to EU not working on MHI2_US_AU57X_S0536

### DISCUSSION THREAD

**Discussion by videomix77** [Mr-MIBonk/M.I.B._More-Incredible-Bash#187] [Q&A]
Hello. I have Audi A6 from 2016, USA version.
Train version: MHI2_US_AU57X_S0536
MU software: 0387

I tried running train convert to EU command, and it failed with message "Train not found in conversion list".
I also see some "Memory fault (core dumped)" messages in the process. Don't know if those are related.
I'm probably missing something obvious.

See [log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/11311250/MU-.log).

**Comment by harman-f** [comment]
Yes, FW is to old, update to latest US 1st

**Reply by videomix77** [reply]
Thank you for replying. I see my problem isn't exactly with M.I.B. but maybe you can help me further.
mib-helper.com suggests MHI2_US_AU57x_K3341_MU1426 as newest, but when updating with normal menu I get "The update is not compatible or are outdated" message and through red menu I get "A variant conflict occurred! (...) Version description variant: Train is blocked". Any other update for MHI2_US_AU57x_* I found on mibsolution.one have the same problem, except MHI2_US_AU57x_K3341_1_AIO_MU1426. With that, I can enter standard update in red menu. Should I use that, or should I activate User Defined SWDL and use other update?

**Reply by harman-f** [reply]
S train is not supported by the updates. 

You have to change metainfo

**Reply by ulianych-bohdan** [reply]
@videomix77 ,did you manage to install this version MHI2_US_AU57x_K3341_1_AIO_MU1426? Or you have some troubles? I have the same problem

**Reply by NoneHere892** [reply]
> S train is not supported by the updates.
> 
> You have to change metainfo

Hi. Would I need to update the metadata file in the update FW or in the installed FW?

**Comment by ulianych-bohdan** [comment]
@videomix77 ,did you manage to install this version MHI2_US_AU57x_K3341_1_AIO_MU1426? Or you have some troubles? I have the same problem

**Comment by justelj1375** [comment]
I tried to update my 2016 Audi A6 (C7.5) MMI from: MHI2_US_AUG11_S0533 MU 0143 --> MHI2_US_AUG57x_K3341_1 MU 1426
Now my MMI is stuck in Emergency Assist Utility reboot loop trying to fix RCC efs and I keep getting file size doesnt match metainfo tag??
Does anyone have the old firmware?
Does anyone have an idea what to do to fix this?
Im stumped and sick to my stomach.....
